policy/resolver.go

```diff
@@ -904,20 +904,20 @@ func (s *LocalServices) policyGroupToJobs(ctx context.Context, group *PolicyGrou
 		if impact == nil {
 			impact = &explorer.Impact{}
 		}
-		if check.Action == explorer.Action_IGNORE {
+		if check.Action == explorer.Action_IGNORE || group.Type == GroupType_IGNORED {
 			impact.Scoring = explorer.ScoringSystem_IGNORE_SCORE
 			impact.Action = explorer.Action_IGNORE
 		}
 
 		cache.global.propsCache.Add(check.Props...)
 
-		if check.Action == explorer.Action_UNSPECIFIED || check.Action == explorer.Action_ACTIVATE {
+		if (check.Action == explorer.Action_UNSPECIFIED || check.Action == explorer.Action_ACTIVATE) && group.Type != GroupType_IGNORED {
```

Comment:
I'm not 100% sure about this.

I think, for security exceptions, "IGNORE" means, that it still get's executed, but not scored. I'd expected "DISABLE" here.

Reply:
this check makes sure that when we have a group with action `IGNORE` we do not call `addCheckJob` but `modifyCheckJob`. The exception groups do not add new checks, they only modify existing ones

Reply:
Thanks for the explanation. Makes sense to me.